README.md

```diff
@@ -44,7 +44,11 @@ A detailed introduction can be found in [introduction.md](./docs/en/tutorials/in
   XRMoCap decomposes the framework into several components, based on which optimization-based and learning-based methods are integrated into one framework. Users can easily prototype a customized multi-view mocap pipeline by choosing different components in configs.
 
 ## News
-
+- 2022-12-21: XRMoCap [v0.7.0](https://github.com/openxrlab/xrmocap/releases/tag/v0.7.0) is released. Major updates include:
+  - Add [mview_mperson_end2end_estimator](https://github.com/openxrlab/xrmocap/blob/main/xrmocap/core/estimation/mview_mperson_end2end_estimator.py) for learning-based method
+  - Add SMPLX support and allow smpl_data initiation in `mview_sperson_smpl_estimator`
+  - Add multiple optimizers, detailed joint weights and priors, grad clipping for better smplify results
+  - Add [mediapipe_estimator](https://github.com/openxrlab/xrmocap/blob/main/xrmocap/human_perception/keypoints_estimation/mediapipe_estimator.py) for human keypoints2d perception
 - 2022-10-14: XRMoCap [v0.6.0](https://github.com/openxrlab/xrmocap/releases/tag/v0.6.0) is released. Major updates include:
   - Add [4D Association Graph](http://www.liuyebin.com/4dassociation/), the first Python implementation to reproduce this algorithm
```

Comment:
Please add [4D Association Graph](http://www.liuyebin.com/4dassociation/) to Supported methods

Reply:
Fixed in [b1687ac](https://github.com/openxrlab/xrmocap/pull/63/commits/b1687ac38cbec17aa3274d7a0c41ea660081c269).